roles/ceph-defaults/vars/main.yml

```diff
@@ -1,3 +1,4 @@
 ---
 ceph_osd_pool_default_size: 3
+ceph_osd_pool_default_min_size: 2
```

Comment:
these value must reflect the defaults value from ceph.

From the doc:
```
osd pool default min size

Description:  Sets the minimum number of written replicas for objects in the pool in order to acknowledge a write operation to the client. If minimum is not met, Ceph will not acknowledge the write to the client, which may result in data loss. This setting ensures a minimum number of replicas when operating in degraded mode.
Type:         32-bit Integer
Default:      0, which means no particular minimum. If 0, minimum is size - (size / 2).
```

unless this default value has changed in nautilus it should be `0`

Reply:
updated.
thank you

Reply:
almost there!
Please squash your commits, thanks!

Reply:
default is still 0 according to latest version of the doc [1] 

[[1]](https://github.com/ceph/ceph/blame/master/doc/rados/configuration/pool-pg-config-ref.rst#L200) `https://github.com/ceph/ceph/blame/master/doc/rados/configuration/pool-pg-config-ref.rst#L200`